extension.js

```diff
@@ -83,48 +106,85 @@ async function activate() {
 	});
 }
 
-async function setColor(color) {
+/**
+ * Checks if the active editor is in a Salesforcxe project
+ * @returns {boolean} true if the active editor is in a Salesforce project
+ */
+async function checkIfInSfProject() {
+	const sfdxConfigFiles = await vscode.workspace.findFiles(
+		OLD_PATH || NEW_PATH,
+		null,
+		1
+	);
+
+	if (sfdxConfigFiles.length > 0) {
+		return true;
+	}
+	return false;
+}
+
+/**
+ *
+ * @param {string} backgroundColor - hex color code for background color
+ * @param {string} [foregroundColor] - hex color code for foreground color (Optional)
+ */
+async function setColor(backgroundColor, foregroundColor) {
 	const config = vscode.workspace.getConfiguration();
+	const settingsScope =
+		config.get('sf-colorg.target.settingsScope') || 'workspace';
```

Comment:
@jlawrence124 we should have by default the 'user' scope if no one is found, to be aligned with the default value for that setting in the package.json file

Reply:
done!
- [Limited `initialCleanup` to only run if `"settingsScope": "user"` ](https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/545f0428fb3c0e4f1ef496ce0c36bda929b1e3c7)

Reply:
done!

https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/32d208c6fea2c6ef6785f57e7f70e57318c32852